### PR TITLE
Personnel XP Standardization

### PR DESCRIPTION
**Pull request by Windchild292**
This PR standardizes how personnel experience is done, removing the need (if there even was one, cause I can't see a reason for it after going over the code) for a separate count for engineering xp. I also used this to make some more formatting changes to improve readability.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/unit/Unit.java [hunk 1]
```diff
@@ -3956,35 +3956,32 @@ private void assignToCrewSlot(Person p, int slot, String gunType, String driveTy
...
+                if (p.getMinutesLeft() < minutesLeft) {
                     minutesLeft = p.getMinutesLeft();
                 }
-                if(p.getOvertimeLeft() < overtimeLeft) {
+
+                if (p.getOvertimeLeft() < overtimeLeft) {
                     overtimeLeft = p.getOvertimeLeft();
                 }
-                if(p.getEdge() < edgeLeft) {
-                    edgeLeft = p.getEdge();
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/unit/Unit.java:3982]: What was this for?

File: MekHQ/src/mekhq/service/MassRepairService.java [hunk 2]
```diff
@@ -1059,7 +1062,7 @@ private static WorkTimeCalculation calculateNewMassRepairWorktime(IPartWork part
         }
 
         WorkTime newWorkTime = partWork.getMode();
-        WorkTime previousNewWorkTime = newWorkTime;
+        WorkTime previousNewWorkTime;
```
Comment (NickAragua) [inline MekHQ/src/mekhq/service/MassRepairService.java:1065]: Does this need a default value still?

Comment (Windchild292) [reply MekHQ/src/mekhq/service/MassRepairService.java:1065]: Nope, it is always overwritten

File: MekHQ/src/mekhq/campaign/unit/Unit.java (hunk 1 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/unit/Unit.java:3982]: Completely unused, so I removed the logic.

**Pull request merged**